packages/zosfiles/src/api/methods/upload/Upload.ts

```diff
@@ -488,12 +488,14 @@ export class Upload {
                                         binary: boolean = false,
                                         localEncoding?: string,
                                         etag?: string,
-                                        returnEtag?: boolean) {
+                                        returnEtag?: boolean,
+                                        responseTimeout?: number) {
```

Comment:
Thanks for also updating the functions that people may be using today but we hope to clean for Zowe V2 😋